README.md

```diff
@@ -41,29 +41,29 @@ For more details on initializing Helm, [read the Helm docs](https://helm.sh/docs
 
 2. Then you have several options to setup OpenFunction, you can choose to:
 
-   - Install all components:
-      ```shell
-      kubectl create namespace openfunction
-      helm install openfunction openfunction/openfunction -n openfunction
-      ```
-   
-   - Install Serving only (without build):
-      ```shell
-      kubectl create namespace openfunction
-      helm install openfunction --set global.ShipwrightBuild.enabled=false --set global.TektonPipelines.enabled=false openfunction/openfunction -n openfunction
-      ```
-   
-   - Install Knative sync runtime only:
-      ```shell
-      kubectl create namespace openfunction
-      helm install openfunction --set global.Keda.enabled=false openfunction/openfunction -n openfunction
-      ```
-   
-   - Install OpenFunction async runtime only:
-      ```shell
-      kubectl create namespace openfunction
-      helm install openfunction --set global.Contour.enabled=false  --set global.KnativeServing.enabled=false openfunction/openfunction -n openfunction
-      ```
+    - Install all components:
```

Comment:
Considering replace these content with https://openfunction.dev/docs/getting-started/installation/#steps-to-install-openfunction-helm-charts to avoid duplicated docs maintenance